warehouse/api.go

```diff
@@ -92,8 +96,14 @@ type UploadAPIT struct {
 	isMultiWorkspace  bool
 }
 
+type ValidateObjectRequestT struct {
```

Comment:
If this is not used, Can we remove this?

Reply:
Done